be/src/storage/schema_change.cpp

```diff
@@ -923,15 +923,17 @@ Status SchemaChangeHandler::process_alter_tablet_v2(const TAlterTabletReqV2& req
 }
 
 Status SchemaChangeHandler::_do_process_alter_tablet_v2(const TAlterTabletReqV2& request) {
-    TabletSharedPtr base_tablet = StorageEngine::instance()->tablet_manager()->get_tablet(request.base_tablet_id);
+    ASSIGN_OR_RETURN(TabletSharedPtr base_tablet,
```

Comment:
According to your new implementation, if `TabletManager::get_tablet` returns an error, it may mean that the tablet does not exist or that there are other unexpected errors. You need to judge:
1. When the tablet does not exist, should return NotFound or InternallError, i.e, is it ok to use `ASSIGN_OR_RETURN` here.
2. If it's ok to use `ASSIGN_OR_RETURN`,  the following `if (base_tablet == nullptr)` check should be removed.